src/utils/migrations.ts

```diff
@@ -0,0 +1,34 @@
+import { adProvidersMethods, adProvidersMethodsLegacy } from '../advertising/external-ads';
+import { redisClient } from '../redis';
+import { isDefined } from './helpers';
+import logger from './logger';
+
+const DATA_VERSION_STORAGE_KEY = 'data_version';
+
+const migrations = {
```

Comment:
Sorry, it's not about data for the selectors by ads providers because their previous format is incompatible with the current one, and there is no way to make them compatible